app/client/src/layoutSystems/anvil/utils/layouts/update/zoneUtils.ts

```
@@ -86,7 +84,7 @@ export function* addWidgetsToZone(
     zoneWidgetId,
     draggedWidgets,
   );
-  zoneProps.children = updatedWidgets[zoneWidgetId].children;
+  updatedWidgets[zoneWidgetId].children = updatedWidgets[zoneWidgetId].children;
```

Biome

[error] children is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)